backend/.vscode/launch.json

```diff
@@ -13,8 +13,8 @@
             "cwd": "${workspaceFolder}/api",
             "stopAtEntry": false,
             "env": {
-                "ASPNETCORE_ENVIRONMENT": "Development",
-                "ASPNETCORE_URLS": "http://localhost:5000"
+                "ASPNETCORE_ENVIRONMENT": "Local",
```

Comment:
When running locally it requires a number of configuration changes.  This makes it a little simpler to go back and forth.